<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="71754" started-at="2017-09-27T20:30:25Z" finished-at="2017-09-27T20:31:37Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="71754" started-at="2017-09-27T20:30:25Z" finished-at="2017-09-27T20:31:37Z">
      <class name="com.zopa.Tests.SignUp_SadPath">
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.zopa.Tests.SignUp_SadPath@6a41eaa2]" name="beforeMethod" is-config="true" duration-ms="16617" started-at="2017-09-27T14:30:25Z" finished-at="2017-09-27T14:30:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="PhoneNumberValidation()[pri:0, instance:com.zopa.Tests.SignUp_SadPath@6a41eaa2]" name="PhoneNumberValidation" duration-ms="5690" started-at="2017-09-27T14:30:42Z" finished-at="2017-09-27T14:30:48Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.zopa.Tests.SignUp_SadPath.PhoneNumberValidation(SignUp_SadPath.java:45)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PhoneNumberValidation -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.zopa.Tests.SignUp_SadPath@6a41eaa2]" name="tearDown" is-config="true" duration-ms="1258" started-at="2017-09-27T14:30:48Z" finished-at="2017-09-27T14:30:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=PhoneNumberValidation status=FAILURE method=SignUp_SadPath.PhoneNumberValidation()[pri:0, instance:com.zopa.Tests.SignUp_SadPath@6a41eaa2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.zopa.Tests.SignUp_SadPath@6a41eaa2]" name="beforeMethod" is-config="true" duration-ms="13733" started-at="2017-09-27T14:30:49Z" finished-at="2017-09-27T14:31:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="PostCodeValidation()[pri:0, instance:com.zopa.Tests.SignUp_SadPath@6a41eaa2]" name="PostCodeValidation" duration-ms="5829" started-at="2017-09-27T14:31:03Z" finished-at="2017-09-27T14:31:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PostCodeValidation -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.zopa.Tests.SignUp_SadPath@6a41eaa2]" name="tearDown" is-config="true" duration-ms="126" started-at="2017-09-27T14:31:08Z" finished-at="2017-09-27T14:31:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=PostCodeValidation status=SUCCESS method=SignUp_SadPath.PostCodeValidation()[pri:0, instance:com.zopa.Tests.SignUp_SadPath@6a41eaa2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.zopa.Tests.SignUp_SadPath@6a41eaa2]" name="beforeMethod" is-config="true" duration-ms="19953" started-at="2017-09-27T14:31:08Z" finished-at="2017-09-27T14:31:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="emailVerification()[pri:0, instance:com.zopa.Tests.SignUp_SadPath@6a41eaa2]" name="emailVerification" duration-ms="8384" started-at="2017-09-27T14:31:28Z" finished-at="2017-09-27T14:31:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- emailVerification -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.zopa.Tests.SignUp_SadPath@6a41eaa2]" name="tearDown" is-config="true" duration-ms="139" started-at="2017-09-27T14:31:37Z" finished-at="2017-09-27T14:31:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=emailVerification status=SUCCESS method=SignUp_SadPath.emailVerification()[pri:0, instance:com.zopa.Tests.SignUp_SadPath@6a41eaa2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.zopa.Tests.SignUp_SadPath -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
